docs/attribute_analysis/drafts/license_appendix.md

```diff
@@ -0,0 +1,301 @@
+# License Field Analysis Appendix
+
+This is an extraction from the https://packages.ecosyste.ms/ service, analyzing the top 9 registries by package count and examining the most commonly used licenses across all active packages. This analysis uses the raw `licenses` field as reported by each package registry, with the analysis covering both the general package population and packages marked as critical infrastructure.
+
+Code for this script lives here: https://github.com/ecosyste-ms/packages/blob/main/lib/tasks/license_analysis.rake
+
+## Global Analysis
+Total packages: 9,671,935
+
+### Top licenses values:
+| License | Count | Percentage |
+|---------|-------|------------|
+| MIT | 3,031,054 | 31.34% |
+| '' | 1,919,350 | 19.84% |
+| NULL | 1,227,701 | 12.69% |
+| ISC | 1,109,627 | 11.47% |
+| Apache-2.0 | 490,475 | 5.07% |
+| MIT License | 152,735 | 1.58% |
+| BSD-3-Clause | 125,015 | 1.29% |
+| The Apache Software License, Version 2.0 | 88,919 | 0.92% |
+| GPL-3.0 | 73,014 | 0.75% |
+| Apache-2.0,Apache-2.0 | 61,429 | 0.64% |
+| Apache License, Version 2.0 | 52,548 | 0.54% |
+| GPL-2.0-or-later | 46,678 | 0.48% |
+| The Apache License, Version 2.0 | 45,312 | 0.47% |
+| GPL-3.0-or-later | 41,066 | 0.42% |
+| MIT OR Apache-2.0 | 39,979 | 0.41% |
+| BSD | 36,975 | 0.38% |
+| BSD-2-Clause | 36,583 | 0.38% |
+| MPL-2.0 | 33,983 | 0.35% |
+| Apache 2.0 | 26,934 | 0.28% |
```

Comment:
There is a few "Apache-2.0" Variations listed, can we consolidate into single value? it seems others are not duplicated, Assuming BSD is not the BSD-2-Clause one, which is expect.